packages/relay/tests/lib/eth.spec.ts

```diff
@@ -3033,6 +3033,55 @@ describe('Eth calls using MirrorNode', async function () {
     expect(gas).to.equal(EthImpl.numberTo0x(defaultGasOverride));
   });
 
+  it('eth_estimateGas with contract revert returns the JsonRpcError', async function () {
```

Comment:
You'd need 1 more test to confirm your fix. One where a 400 returns the `'execution reverted: '` message and you confirm that it doesn't throw but instead returns a static value

Reply:
Done.